pkg/cmd/taskrun/logs_test.go

```diff
@@ -258,8 +258,8 @@ func TestLog_taskrun_logs(t *testing.T) {
 	output, _ := fetchLogs(trlo)
 
 	expectedLogs := []string{
-		"[writefile-step] wrote a file\n",
-		"[nop] Build successful\n",
+		"[output-task : writefile-step] wrote a file\n",
```

Comment:
I don't know why only taskrun's log didn't include task name on log format.

but I think it would be better including task name on log like pipelinerun's log, 
so this PR include it together with refactoring.

Reply:
I think it's because it's implicit? i guess I am good with this, I am just worry if someone (ie.: the vscode tekton plugin use tkn as backend) depends on that output...

Reply:
> I think it's because it's implicit? i guess I am good with this, I am just worry if someone (ie.: the vscode tekton plugin use tkn as backend) depends on that output...

Ahh.. okay, I'll undo it, thanks !!

Reply:
>I think it's because it's implicit? i guess I am good with this, I am just worry if someone (ie.: the vscode tekton plugin use tkn as backend) depends on that output...

+1

Reply:
@chmouel sorry for late, I undid it !!